prow/plugins.yaml

```diff
@@ -69,6 +68,10 @@ approve:
   - bazelbuild
   implicit_self_approve: true
   review_acts_as_approve: true
+- repos:
```

Comment:
Why not just change the name of the repo in the above list?

Reply:
I noticed the pattern was to break out those not in the kubernetes org to separate config (e.g., bazelbuild). Even when the config is the same. I was trying to follow the existing convention.

Reply:
Seems like an antipattern? I don't feel strongly, defer to engprod

Reply:
breaking them out kinda makes sense, at some point we might want seperate prow deployments? it's sorta iffy what should be on this one, for not it's just easiest to use this one for everything.